features/mbedtls/targets/TARGET_NUVOTON/TARGET_NUC472/sha/sha_alt_hw.c

```diff
@@ -260,71 +255,72 @@ void crypto_sha_update_nobuf(crypto_sha_context *ctx, const unsigned char *input
                 }
                 CRPT->SHA_DMACNT = lastblock_size;
                 CRPT->SHA_CTL = sha_ctl_start | CRPT_SHA_CTL_DMALAST_Msk;
```

Comment:
@ccli8 What's the reason the code sets `SHA_CTL` twice in each iteration? Once in the beginning of the loop, and once in each of the branches distinguishing `rmn <= 4` and `islast`.

Reply:
@hanno-arm I remove duplicate configuration in https://github.com/ARMmbed/mbed-os/pull/4925/commits/a5fc5b8dba8b164e21a95cf95fba27dc75c6cf04.